getting-started/aws/sample-helm/values-velocity.yaml

```diff
@@ -1,13 +1,13 @@
 # {velocity.v1.envName} will be translated to unique ID per environment
 
 # bucket_name is prefixed with a random string and the environment name. This is because buckets are globally unique, so this allows us to have a bucket per environment without collisions.
-bucket_name: '{velocity.v1.generate:random([a-z]{1,64})}-{velocity.v1.envName}-my-bucket'
+bucket_name: '{velocity.v1.generate:cloudResourcePrefix(seed=buket)}-{velocity.v1.envName}-my-bucket'
```

Comment:
Typo in seed 
:P even though not matter

Reply:
Y not include env-name in the cloud resource prefix?
Also instead of seed - maybe use - id - to encapsulate the fact it's a seed and it's a bit more readable maybe?

Reply:
changing now the word seed to ID will require a lot of effort, maybe we could explain this in the docs?